nmdc_server/migrations/versions/ceb93b48d6ae_add_field_noes_metadata.py

```diff
@@ -0,0 +1,34 @@
+"""Add field_notes_metadata column to submission_metadata
+
+Revision ID: ceb93b48d6ae
+Revises: 4aee1e10bb24
+Create Date: 2024-11-18 21:49:50.640253
+
+"""
+
+from typing import Optional
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = "ceb93b48d6ae"
+down_revision: Optional[str] = "4aee1e10bb24"
```

Comment:
#1465 was just merged, which introduced a migration, you'll have to update the `Revises:` comment and the `down_revision` identifier here.